static/app/views/settings/components/dataScrubbing/modals/form/index.tsx

```diff
@@ -150,6 +146,7 @@ class Form extends Component<Props<Values, KeysOfUnion<Values>>, State> {
                 onChange={this.handleChange('pattern')}
                 value={values.pattern}
                 onBlur={onValidate('pattern')}
+                aria-label={t('Regular expression')}
```

Comment:
I think it's better to give this a `id` and give the `field` a matching `id`. Then it will be correctly associated with the fields label

Reply:
hm... I am not sure about that... can we talk about it offline? I can update it in a follow-up

Reply:
yeah that's fine

If you give the `Field` an `id` it sets it as the `htmlFor`. It's a bit confusing

https://github.com/getsentry/sentry/blob/dc06981596544c30429b8e36eedd62f444469bd7/static/app/components/forms/field/index.tsx#L101

Then when you give the actual control a field it will link with the label. So you don't need two aria labels

Reply:
This is typically how inputs / other controls get their labels assigned.

Reply:
oh I see! thanks! will update that in a follow-up